vthunder/resource_vthunder_slb_sip.go

```diff
@@ -0,0 +1,98 @@
+package vthunder
+
+//vThunder resource SlbSip
+
+import (
+	"fmt"
+	"util"
+
+	go_vthunder "github.com/go_vthunder/vthunder"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceSlbSip() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSlbSipCreate,
+		Update: resourceSlbSipUpdate,
+		Read:   resourceSlbSipRead,
+		Delete: resourceSlbSipDelete,
+		Schema: map[string]*schema.Schema{
+			"sampling_enable": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"counters1": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "",
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func resourceSlbSipCreate(d *schema.ResourceData, meta interface{}) error {
+	logger := util.GetLoggerInstance()
+	client := meta.(vThunder)
+
+	if client.Host != "" {
+		logger.Println("[INFO] Creating SlbSip (Inside resourceSlbSipCreate) ")
+
+		data := dataToSlbSip(d)
+		logger.Println("[INFO] received V from method data to SlbSip --")
```

Comment:
please make the logs more meaningful. Please replace V with better info.

Reply:
Updated all resources with V in them.